Assets/UnityShell/Editor/Scripts/AutocompleteProvider.cs

```diff
@@ -18,19 +14,28 @@ public class AutocompleteProvider
 		public void SetInput(string input)
 		{
 			handleCount++;
-			new Thread(() =>
+			new Thread((threadStart) =>
 			{
 				int handle = handleCount;
 
 				if(!string.IsNullOrEmpty(input))
 				{
 					string prefix;
+#if NET_4_6 || NET_STANDARD_2_0
+					var evaluator = new Evaluator(new CompilerContext(new CompilerSettings(), new ConsoleReportPrinter()));
```

Comment:
As this is a new Evaluator, does Autocomplete still work exactly the same, as usings that are in ShellEveluator are not applied to this evaluator?

Reply:
@VoxelBoy ?

Reply:
Haven't had the time, sorry. I'll try looking into this today.

Reply:
Fixed. In my testing, autocomplete now comes up faster for 4.x runtime than 3.5.